README.rst

```diff
@@ -3,3 +3,20 @@ ts_MTMount
 ##########
 
 ``ts_MTMount`` is a CSC that controls the Simonyi Survey Telescope main axes and related components.
+
+`Documentation <https://ts-mtmount.lsst.io>`_
+
+The package is compatible with the `eups <https://github.com/RobertLuptonTheGood/eups>`_ package management system and ``scons`` build system.
+Assuming you have the basic Vera C. Rubin LSST DM stack installed you can do the following, from within the package directory:
+
+* ``setup -r .`` to setup the package and dependencies.
```

Comment:
I was only able to make mine work by running `setup -k -r .` otherwise it would complain about not being able to find the ts_hexrotcomm package. I pasted the relevant terminal text below.

[saluser@537999b4d9b6 ts_hexrotcomm]$ setup -r .
[saluser@537999b4d9b6 ts_hexrotcomm]$ cd ..
[saluser@537999b4d9b6 gitdir]$ ls
docker                       ts_ATDomeTrajectory  ts_sal
ospl-info.log                ts_ATMCSSimulator    ts_salobj
robotframework_salgenerator  ts_Dockerfiles       ts_scriptqueue
templates                    ts_hexrotcomm        ts_simactuators
test_max_speed_slew.py       ts_MTDomeTrajectory  ts_standardscripts
ts_ATDome                    ts_MTMount           ts_xml
[saluser@537999b4d9b6 gitdir]$ cd ts_MTMount/
[saluser@537999b4d9b6 ts_MTMount]$ ls
bin         COPYRIGHT  LICENSE  README.rst  setup.cfg  ups
config.log  doc        python   SConstruct  tests
[saluser@537999b4d9b6 ts_MTMount]$ setup -r .
setup: in file /home/saluser/gitdir/ts_MTMount/ups/ts_MTMount.table: Product ts_hexrotcomm not found
[saluser@537999b4d9b6 ts_MTMount]$ setup -k -r .
[saluser@537999b4d9b6 ts_MTMount]$ scons

Reply:
Interesting. ts_hexrotcomm is not present in lsstts/develop-env:b41. @tribeiro is it in more recent version? I need it for MTMount.